tests/unit/views/graph.vue.spec.js

```diff
@@ -116,215 +116,215 @@ describe('Graph view', () => {
     )
   })
 
-  it('it gets families', async () => {
-    const wrapper = mount(Graph, {
-      shallow: true,
-      global: {
-        plugins: [store],
-        mocks: { $workflowService }
-      },
-      props: {
-        workflowName: 'one',
-      },
-      computed: {
-        workflows () {
-          return workflows
-        },
-        namespaces () {
-          return namespaces()
-        },
-        cylcTree () {
-          return cylcTree
-        }
-      }
-    })
+  // it('it gets families', async () => {
```

Comment:
Whoops!

Note, you can use `.only` during development to only run targeted tests, e.g:

```js
it.only('it works', ....
```